mailpoet/assets/css/src/components-automation-editor/_step.scss

```diff
@@ -33,3 +33,19 @@
 .mailpoet-automation-editor-step-subtitle {
   color: inherit;
 }
+
+.colored-icon {
+  box-sizing: content-box;
+  padding: 10px;
+  position: relative;
+
+  .colored-icon-background {
+    border-radius: 50%;
+    height: 100%;
+    left: 0;
+    opacity: .1;
```

Comment:
Ahh, this probably is why the email icon background has a different color than in Figma. I think we should keep full opacity to reflect the exact used color, if possible. We'll need two colors (foreground and background).

Reply:
Main commit to address this is [d403d7a](https://github.com/mailpoet/mailpoet/pull/4275/commits/d403d7ae91f2c6db2b81fbd35d11df307b507fa5)